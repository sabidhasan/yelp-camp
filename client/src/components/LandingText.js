import React from 'react'

const LandingText = () => {
  return (
    <div className='LandingText'>
      <section className='LandingText__section'>
        <h1 className='LandingText__title'>Truly Canadian</h1>
        <p className='LandingText__text'>We list campgrounds from across Canada - from day trips
         in Ontario, to secluded sites in the Yukon.</p>
      </section>
      <section className='LandingText__section'>
        <h1 className='LandingText__title flex-center'>Save Time</h1>
        <p className='LandingText__text'>Detailed information for each campground means you spend
        less time searching, and more time camping!</p>
      </section>
      <section className='LandingText__section'>
        <h1 className='LandingText__title'>Unfiltered Reviews</h1>
        <p className='LandingText__text'>We offer unbiased reviews for each campground so you can truly
        decide what's worth visiting in your limited summer.</p>
      </section>
    </div>
  )
}

export default LandingText
