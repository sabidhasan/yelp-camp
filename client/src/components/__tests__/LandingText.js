import React from 'react'
import toJson from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import LandingText from '../LandingText'

configure({ adapter: new Adapter() });

describe('<LandingText />', () => {
  const component = shallow(<LandingText />)

  it('Component should be defined', () => {
    expect(component).toBeDefined();
  });

  it('LandingText component snapshot matches what is expected', () => {
    expect(toJson(component)).toMatchSnapshot();
  })
})
