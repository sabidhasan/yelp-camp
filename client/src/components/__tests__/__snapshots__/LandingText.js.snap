// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingText /> LandingText component snapshot matches what is expected 1`] = `
<div
  className="LandingText"
>
  <section
    className="LandingText__section"
  >
    <h2
      className="LandingText__title flex-center"
    >
      Truly Canadian
    </h2>
    <p
      className="LandingText__text"
    >
      We list campgrounds from across Canada - from day trips in Ontario, to secluded sites in the Yukon.
    </p>
  </section>
  <section
    className="LandingText__section"
  >
    <h2
      className="LandingText__title flex-center"
    >
      Save Time 
      <i
        aria-hidden="true"
        className="fas fa-clock"
      />
    </h2>
    <p
      className="LandingText__text"
    >
      Detailed information for each campground means you spend less time searching, and more time camping!
    </p>
  </section>
  <section
    className="LandingText__section"
  >
    <h2
      className="LandingText__title flex-center"
    >
      Unfiltered Reviews 
      <i
        aria-hidden="true"
        className="fas fa-filter"
      />
    </h2>
    <p
      className="LandingText__text"
    >
      We offer unbiased reviews for each campground so you can truly decide what's worth visiting in your limited summer.
    </p>
  </section>
</div>
`;
